source/apis/routes.py

```diff
@@ -20,12 +39,20 @@ async def say_hello(name: str):
 
 
 @router.get("/generate-doc")
-async def generate_document(n_pages=10, font_size=12, fontype="font1"):
-    pdf_buffer = generate_pdf(text, n_pages, font_size, fontype)
-    return Response(content=pdf_buffer.getvalue(), media_type="application/pdf",
-                    headers={"Content-Disposition": "attachment; filename=generated_doc.pdf"})
+async def generate_document(
+        fonttype: str = Query(..., description="Font type"),
+        n_pages: int = Query(10, description="Number of pages"),
+        font_size: int = Query(12, description="Font size"),
+):
+    selected_font = fitz.Font(fontfile=fonts.get(fonttype))
```

Comment:
if you only need the Font class then use from fitz import Font, importing the whole package adds more strain on the application. ALways import just what you need!